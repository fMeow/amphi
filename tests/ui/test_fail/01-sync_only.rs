use amphi::amphi;

#[amphi(sync_only)]
mod amphi{
    pub async fn my_fn() -> bool {
        true
    }
}

fn main() {
    use self::asynchronous::my_fn;
}
