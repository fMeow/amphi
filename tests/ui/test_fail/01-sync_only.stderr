error[E0432]: unresolved import `self::asynchronous`
  --> $DIR/01-sync_only.rs:12:15
   |
12 |     use self::asynchronous::my_fn;
   |               ^^^^^^^^^^^^ maybe a missing crate `asynchronous`?
